### PR TITLE
Smoke test Missing buttons

### PR DESCRIPTION
**Pull request by ozdemir08**
RTMP: IsRoomConnected, GetInvitation
PGSClient: SetGravityForPopups
Signin: Different options

### CODE REVIEW

File: samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs [hunk 1]
```diff
@@ -271,12 +266,38 @@ internal void ShowNotAuthUi()
...
             this.DrawStatus();
-            if (GUI.Button(this.CalcGrid(0, 1), "Authenticate"))
+            if (GUI.Button(this.CalcGrid(0, 1), "Authenticate - Simple"))
             {
-                this.DoAuthenticate();
+                this.DoAuthenticate(new PlayGamesClientConfiguration.Builder().Build());
             }
-
-            if (GUI.Button(this.CalcGrid(1, 1), "Nearby Connections"))
+            else if (GUI.Button(this.CalcGrid(1, 1), "Authenticate - Token"))
```
Comment (olehkuznetsov) [inline samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs]: - ID Token

File: samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs [hunk 2]
```diff
@@ -271,12 +266,38 @@ internal void ShowNotAuthUi()
...
+            {
+                this.DoAuthenticate(new PlayGamesClientConfiguration.Builder()
+                    .RequestIdToken()
+                    .Build());
+            }
+            else if (GUI.Button(this.CalcGrid(0, 2), "Authenticate - Authcode"))
```
Comment (olehkuznetsov) [inline samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs]: - Server Auth Code

File: samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs [hunk 3]
```diff
@@ -271,12 +266,38 @@ internal void ShowNotAuthUi()
...
+            {
+                this.DoAuthenticate(new PlayGamesClientConfiguration.Builder()
+                    .RequestServerAuthCode(false)
+                    .Build());
+            }
+            else if (GUI.Button(this.CalcGrid(1, 2), "Authenticate - Saved Game"))
```
Comment (olehkuznetsov) [inline samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs]: - Enable Saved Games

File: samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs [hunk 4]
```diff
@@ -271,12 +266,38 @@ internal void ShowNotAuthUi()
...
+            else if (GUI.Button(this.CalcGrid(0, 3), "Authenticate - Full"))
+            {
+                this.DoAuthenticate(new PlayGamesClientConfiguration.Builder()
+                    .RequestIdToken()
+                    .RequestEmail()
+                    .RequestServerAuthCode(false)
+                    .EnableSavedGames()
+                    .Build());
+            }
+            else if (GUI.Button(this.CalcGrid(1, 3), "Nearby Connections"))
```
Comment (olehkuznetsov) [inline samples/SmokeTest/Source/Assets/SmokeTest/MainGui.cs]: Let's add another one. - Hide Popups
.EnableHidePopups()

**Comment by ozdemir08** [comment]
Buttons are renamed and Hide popups signin option is added.

**Pull request merged**